i created tribute project

fatiabdulahi opened the pull request:
it's the tribute project .I work with html and css

fatiabdulahi committed:
i created tribute project

fatiabdulahi commented:
@Saqibi4213 dear Mehrea could you review my repository .and shear you'r opinion .thank you

fatiabdulahi committed:
hope to work

fatiabdulahi committed:
edit 1

fatiabdulahi committed:
scond edite

fatiabdulahi commented:
@ahzamir hello sir, could you review my project finally I solved all linters error.

Saqibi4213 commented:
you done have REAME.md file you should have professional readme file and share your project demo also 
and your code is not complete if you created tribute page your code is incomplete try to complete the requirement and instruction @fatiabdulahi

fatiabdulahi commented:
@Saqibi4213 hi dear partner please review my project and shear your opinean

ahzamir commented:
@fatiabdulahi 

- Your partner gave you feedback yesterday and you still have not implemented it yet.

fatiabdulahi committed:
hope to worke

fatiabdulahi committed:
Update README.md



 Tribute page 🚀

## Project Description 📝

> This is a tribute page for getting information about Dr Sima Samar.




## Demo 📸
https://github.com/fatiabdulahi/TRIBUTE-F-REPO/blob/feature/tribute_demo.png


## Technologies Used 🛠️

- HTML
- CSS

## Installation 💻

Folow these steps to set up the project on your local machine 
Ensure you have the following software installed on your machine

bash
#git clone


bash
#git@github.com:fatiabdulahi/TRIBUTE-F-REPO/.git


## Usage 🎯

you can access to my project through git clone 
-git clone the HTTP or My SSH code link from the repository you wat to clone 

 
#git clone


## Features ⭐

1: General Information
Name:
Email:
Age:
Gender:
2: Kitchen Use and Preferences
3: Food Taste Preferences
4: Suggestions and Feedback
## Author 👩‍💻


Fatima Abdullahi

- Email: [ fatima333abdullahi@gmail.com)
- 


## Contributing 🤝

We welcome contributions from the community! Please follow the guidelines below to contribute to our project.
<br />
1: Fork the Repository: Click the "Fork" button at the top right of the repository page to create a copy of the repository on your GitHub account.
<br />
2: Clone Your Fork: Clone your forked repository to your local machine using the following command:git clone https://github.com/yourusername/yourprojectname.git
<br />
3: Create a Branch: Create a new branch for your work to ensure your changes are isolated from the main branch:git checkout -b feature-or-bugfix-description
<br />
4: Make Your Changes: Implement your changes, whether fixing a bug or adding a new feature. Be sure to follow the project's coding standards and guidelines.
<br />
5: Commit Your Changes: Commit your changes with a descriptive commit message:git add .
git commit -m "Description of the changes"
<br />
6: Push to Your Fork: Push your changes to your forked repository:git push origin feature-or-bugfix-description
<br />
7: Submit a Pull Request: Go to the original repository and submit a pull request from your forked repository. Provide a clear and concise description of your changes and why they should be merged..

fatiabdulahi committed:
Merge branch 'develop' into feature

fatiabdulahi commented:
@Saqibi4213 hello dear Mehri I edeat my project could you reivew and shear your opinion

fatiabdulahi commented:
@ahzamir hello sir i hope you are doing will  I edit read me could you reviow againe

Saqibi4213 commented:
@fatiabdulahi  dear your readme is not complete and professional

fatiabdulahi committed:
Update README.md

fatiabdulahi committed:
Update README.md


 Tribute page 🚀

## Project Description 📝

> This is a tribute page for getting information about Dr Sima Samar.




## Demo 📸
https://github.com/fatiabdulahi/TRIBUTE-F-REPO/blob/feature/tribute_demo.png
![Alt text](tribute_demo.png)


## Technologies Used 🛠️

- HTML
- CSS

## Installation 💻

Folow these steps to set up the project on your local machine 
Ensure you have the following software installed on your machine

bash
#git clone


bash
#git@github.com:fatiabdulahi/TRIBUTE-F-REPO/.git


## Usage 🎯

you can access to my project through git clone 
-git clone the HTTP or My SSH code link from the repository you wat to clone 

 
#git clone


## Features ⭐

1: General Information
Name:
Email:
Age:
Gender:
2: Kitchen Use and Preferences
3: Food Taste Preferences
4: Suggestions and Feedback
## Author 👩‍💻


Fatima Abdullahi

- Email: [ fatima333abdullahi@gmail.com)
- https://www.linkedin.com/in/fatima-abdullahi-6288662b2?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app


## Contributing 🤝

We welcome contributions from the community! Please follow the guidelines below to contribute to our project.
<br />
1: Fork the Repository: Click the "Fork" button at the top right of the repository page to create a copy of the repository on your GitHub account.
<br />
2: Clone Your Fork: Clone your forked repository to your local machine using the following command:git clone https://github.com/yourusername/yourprojectname.git
<br />
3: Create a Branch: Create a new branch for your work to ensure your changes are isolated from the main branch:git checkout -b feature-or-bugfix-description
<br />
4: Make Your Changes: Implement your changes, whether fixing a bug or adding a new feature. Be sure to follow the project's coding standards and guidelines.
<br />
5: Commit Your Changes: Commit your changes with a descriptive commit message:git add .
git commit -m "Description of the changes"
<br />
6: Push to Your Fork: Push your changes to your forked repository:git push origin feature-or-bugfix-description
<br />
7: Submit a Pull Request: Go to the original repository and submit a pull request from your forked repository. Provide a clear and concise description of your changes and why they should be merged..

fatiabdulahi commented:
@Saqibi4213 hello dear Meheri I edite my read me please revewo agean

Saqibi4213 commented:
- 1 : you dont have linter in your repository set your linter config 
- 2 :  your readme file is not professional still need some requirements 
![Screenshot (31)](https://github.com/fatiabdulahi/TRIBUTE-F-REPO/assets/168208262/08a271b7-7289-4a88-b8f5-2b57b03252ba)
```
#git clone 
```
for example do this not like the screenshot that did 

- 3 : you did not deployed your project and you should add  after deployed get live link add it to your readme 
- 
![Screenshot (32)](https://github.com/fatiabdulahi/TRIBUTE-F-REPO/assets/168208262/9472f9f9-2718-415c-8a84-c66d620f2af0)
- in this author : you should add your depoly live link and also add your Linkedln and facebook link 
@fatiabdulahi
